src/cio.ts

```diff
@@ -89,15 +90,26 @@ export async function identifyUserStreak({
   }
 }
 
-export async function identifyUser({
-  con,
-  cio,
-  user,
-}: {
-  con: DataSource;
-  cio: TrackClient;
-  user: ChangeObject<User>;
-}): Promise<void> {
+export const generateIdentifyObject = async (
+  con: ConnectionManager,
+  user: ChangeObject<User>,
+) => {
+  const { id } = user;
+  const changed = JSON.parse(JSON.stringify(user));
```

Comment:
Not sure if we have a util function that can turn regular objects into `ChangeObject` type.

Reply:
structuredClone maybe?

Reply:
Right. Yeah, I think that's better than json-parse-stringify.